app/hydrator/services/create/stores/config-store.js

```diff
@@ -1260,6 +1280,15 @@ class HydratorPlusPlusConfigStore {
     };
 
     let removeFromUserDrafts = (adapterName) => {
+      // save schedule on deploy
+      this.myPipelineApi.createSchedule(
```

Comment:
Why are we putting the createSchedule code in `removeFromUserDrafts`? It feels a bit counterintuitive

Reply:
I think originally I was trying to do it outside of this func but it didn't work. I'll just rename the function